src/coreComponents/constitutive/fluid/multifluid/CO2Brine/functions/PVTFunctionHelpers.hpp

```diff
@@ -204,13 +204,13 @@ initializePropertyTable( string_array const & inputParameters,
 
     real64 const TStart = units::convertKToC( stod( inputParameters[5] ) );
     real64 const TEnd = units::convertKToC( stod( inputParameters[6] ) );
-    real64 const dT = stod( inputParameters[7] );
+    real64 const dT = stod( units::convertKToC( inputParameters[7] ));
```

Comment:
```suggestion
    real64 const dT = stod( inputParameters[7] );
```
We don't need to convert the `dT` since it's a temperature difference which should be the same in C or K.